index.md

```diff
@@ -84,9 +111,17 @@ This pipeline will produce a number of directories and files:
 - **multisample.vcf.gz**: A multisample VCF file with the variantcalling
 results.
 
-## About
-This pipeline is part of [BioWDL](https://biowdl.github.io/)
-developed by [the SASC team](http://sasc.lumc.nl/).
+## Scattering
+The pipeline performs scattering to speed up analysis on grid computing
+clusters. This means that the reference genome will be split into regions of
+roughly equal size (see the `scatterSize` inputs). For certain steps each of
+these regions will be analyzed in separate jobs, allowing them to be processed
+in parallel.
+
```

Comment:
"The pipeline performs scattering to speed up analysis on grid computing
developed by [the SASC team](http://sasc.lumc.nl/).
clusters. This means that the reference genome will be split into regions of
roughly equal size (see the `scatterSize` inputs). For certain steps each of
these regions will be analyzed in separate jobs, allowing them to be processed
in parallel." is not really correct.

The pipeline performs scattering to speed up analysis on a local HPC cluster. This means that during the alignment and variant calling, samples will be divided into chunks of roughly equal size and each chunk will be analyzed in separate parallel jobs.

Reply:
There are actually to types of scattering performed. The first is used for alignment and QC. The FastQ files are split into chunks and these are run in parallel jobs. This doesn't work for variant calling, though, since you need to have all the data for a certain position in the same job. Instead, the chunks from alignment get merged back together and scattering is performed as I described here.